internal/ingress/controller/controller.go

```diff
@@ -1224,6 +1224,11 @@ func mergeAlternativeBackends(ing *ingress.Ingress, upstreams map[string]*ingres
 
 			altUps := upstreams[upsName]
 
+			if altUps == nil {
+				klog.Errorf("alternative backend %s has already be removed", upsName)
```

Comment:
Is this really an error?  Maybe warn here?

Reply:
Just tell me. I can change it in a bit!

Reply:
let's make it a warning and fix the typo in the message:

> alternative backend %s has already been removed

Reply:
Done